language/Portuguese_Brazil/strings.po

```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
 
 #: src/frontend/mame/ui/custui.cpp:157 src/frontend/mame/ui/dirmenu.cpp:38
 msgid "Language"
-msgstr "Linguagem"
+msgstr "Idioma"
 
 #: src/frontend/mame/ui/custui.cpp:161
 msgid "Show side panels"
 msgstr "Mostrar painéis laterais"
 
 #: src/frontend/mame/ui/custui.cpp:173
 msgid "Custom UI Settings"
-msgstr "Configurações Customizadas do UI"
+msgstr "Configurações Personalizadas da IU"
```

Comment:
**IU**  meaning user interface (interface de usuário) sounds pretty weird. Maybe we should use the word "Interface", but that would result in a pretty long string: "Configurações Personalizadas da Interface". For that reason, a shorter, but nice, alternative could be "Personalizações da Interface", roughly meaning "UI customizations"